app/routes/_index.tsx

```diff
@@ -29,3 +29,29 @@ export default function Index() {
     </>
   );
 }
+
+export function ErrorBoundary() {
+  const error = useRouteError();
+
+  if (isRouteErrorResponse(error)) {
+    return (
+      <div>
```

Comment:
Should we use the same `ErrorMessage` component as above here as well?

Reply:
We can use) I suggest deleting this ErrorBoundary component from index.tsx until we handle errors here differently. Hope it won't lead to performance issues.